manticore/core/state.py

```diff
@@ -248,5 +249,4 @@ def invoke_model(self, model):
 
         :param callable model: Model to invoke
         '''
-        # TODO(mark): this can't support varargs core models!
-        self.platform.invoke_model(model, prefix_args=(self,))
+        self.platform.invoke_model(model, prefix_args=(self,), variadic=isvariadic(model))
```

Comment:
If we're standardizing on `@variadic` attribute, do we want to remove the `variadic` parameter to `invoke_model`? It sounds like we'll always pass `isvariadic()` to it anyway, might as well move that check inside and simplify the interface.